typegate/tests/random/injection/random_injection.ts

```diff
@@ -31,6 +31,9 @@ typegraph("random_injection", (g: any) => {
   const pub = Policy.public();
   const deno = new DenoRuntime();
 
+  // Configure random injection seed value or the default will be used
+  g.configureRandomInjection(1);
```

Comment:
Nice!
I would suggest adding more context to the parameters, like `g.configureRandomInjection({seed: 1});` wdyt

Reply:
Good point! I think it's good to add the parameters to more context and semantics to the code usage, will push an update

Reply:
Done